docs/sketch-specification.md

```diff
@@ -7,7 +7,9 @@ The programs that run on Arduino boards are called "sketches". This term was inh
 
 The sketch root folder name and code file names must start with a basic letter (`A`-`Z` or `a`-`z`), number (`0`-`9`)
 [<sup>1</sup>](#leading-number-note), or underscore (`_`) [<sup>2</sup>](#leading-underscore-note) followed by basic
-letters, numbers, underscores, dots (`.`) and dashes (`-`). The maximum length is 63 characters.
+letters, numbers, underscores, dots (`.`) and dashes (`-`). The maximum length is 63 characters. The sketch name cannot
+end with a dot (`.`) and cannot be a
+[reserved name](https://learn.microsoft.com/en-us/windows/win32/fileio/naming-a-file#naming-conventions).
```

Comment:
```suggestion
[reserved name](https://learn.microsoft.com/windows/win32/fileio/naming-a-file#naming-conventions).
```
This change will cause the link to automatically redirect to the version of the page that is localized according to the user's configured locale. For example, if your browser is configured for Italian language, the suggested URL will redirect to this page:

https://learn.microsoft.com/it-it/windows/win32/fileio/naming-a-file#naming-conventions

With the previous URL, the version of the page localized for the en-us locale is forced regardless of the user's configured locale.